thesis-de/content-de.tex

```diff
@@ -164,7 +164,48 @@ \subsubsection{Frontends}
 \mr%
 
 \section{Projektmanagement}
-agil, sprints, userstories
+Die Liste an verfügbaren Managementmethoden ist lang. Methoden wie ``PRINCE2''
+oder ``Lean Management'' kommen aus dem Bereich des Projektmanagements und sind
+seit vielen Jahren auch im Bereicht der Softwareentwicklung vertreten. Hinzu
+kommen Vorgehensmodelle aus der Softwareentwicklung selbst, wie das
+Wasserfallmodell oder das Spiralmodell. In den letzten Jahren gab es einen
+Wandel hin zu agiler Entwicklung. Es hat sich gezeigt, dass sich in der
+schnelllebigen Informationswelt Anforderungen an Software regelmäßig ändern,
+auch während der Entwicklungs- und Planungsphase und nicht erst im späteren
+Betrieb. Außerdem gibt es in den meisten Projekten unvorhergesehne
+Zwischenfälle. Dazu gehören unter anderem:
+
+\begin{outline}
+  \1 Sicherheitslücken in verwendeten Bibliotheken werden entdeckt. Diese
+  müssen oftmals aufwendig aktualisiert werden.
+  \1 Die Zeiteinschätzung für die Implementierung von Funktionen oder die
+  Behebung von Fehlern benötigt wesentlich mehr oder weniger Zeit als
+  geschätzt.
+\end{outline}
+
+Die Techniken und Vorgehensweisen der agilen Softwareentwicklung versuchen dem
+entgegenzubeugen. Sie zeichnen sich dadurch aus, das sie:
+
+\begin{outline}
+  \1 Iterativ arbeiten
+  \1 Einen sehr geringen Bürokratie Mehraufwand besitzen
+  \1 Sich flexibel an das Projekt und an Änderungen anpassen
+\end{outline}
+
+Die Firma Puppet benutzt mit großen Erfolg agile Softwareentwicklung seit
+Jahren für Ihre Produkte. Sie steht dem Projektteam mit hilfreichen Tips und
+Schulungen zur Seite. ``Agile Softwareentwicklung'' gilt als Oberbegriff für
+alle Techniken und Vorgehensweisen in dem Bereich. Die
+Softwareentwicklungsmodelle Kanban und Scrum nutzen Teile der agilen
+Softwareentwicklung. Das Projektteam entschied sich für die Nutzung agiler
+Projektmanagement Methoden da diese zur Zeit am verbreitesten sind in der
```

Comment:
"Methoden da" -> Methoden, da
Allerdings hört sich der komplette Satz komisch an:
"Das Projektteam entschied sich für die Nutzung agiler
Projektmanagement Methoden da diese zur Zeit am verbreitesten sind in der Wirtschaft"
-> Das Projektteam entschied sich für die Nutzung agiler Projektmanagement Methoden, da diese in der Wirtschaft aktuell am verbreitesten sind. <- eventuell so